analytics_data_api/v0/tests/views/test_courses.py

```diff
@@ -652,6 +652,90 @@ def test_get_404(self):
         self.assertEquals(response.status_code, 404)
 
 
+class CourseUsersProblemDataTests(DemoCourseMixin, TestCaseWithAuthentication):
+    def _get_data(self, user_id=None):
+        return self.authenticated_get('/api/v0/courses/{}/users/{}/problem_data/'.format(self.course_id, user_id))
+
+    def test_get_problem_data(self):
+        user_id = 111
+        G(
+            models.UserProblemWeeklyData,
+            id=1,
+            week_ending="2015-08-20",
```

Comment:
This should be probably changed to a datetime object (for the mock input, but the expected output below should be unchanged).